CHANGELOG.md

```diff
@@ -28,6 +28,8 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 - Fixed positioing of `Popup` in scrollable container @layershifter ([#1341](https://github.com/stardust-ui/react/pull/1341))
 
 ### Features
+- Add `accordionBehavior`, `accordionTitleBehavior` and `accordionContentBehavior` @silviuavram ([#1322](https://github.com/stardust-ui/react/pull/1322)) 
+- Add `expanded` prop to `Accordion` @silviuavram ([#1322](https://github.com/stardust-ui/react/pull/1322)) 
```

Comment:
why not make it as one feature the same as a PR's name - Add keyboard navigation and screen reader support for Accordion? It might make more sense for the end users

Reply:
It's a good idea, will change. But shouldn't we emphasize that we added some props as well? As a separate entry apart from kb navigation and screen reader support?